Standards/scs-0102-w1-image-metadata-implementation-testing.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "SCS Image Metadata: Implementation and Testing Notes"
+type: Supplement
+track: IaaS
+status: Proposal
+supplements:
+  - scs-0102-v1-image-metadata.md
+---
+
+## Introduction
+
+The standard defines a set of metadata properties for public images, that need to be set in order
+to have an SCS-compliant IaaS setup.
+
+## Implementation notes
+
+The [OpenStack Image Manager from OSISM](https://github.com/osism/openstack-image-manager)
+will create a set of images from a "spec file" provided by the user, which can also set the required properties
+for these images.
+
+## Automated tests
+
+### Images sample
+
+Some checks need to be performed on a live instance. All publicly available images on this instance
+will be checked for either only the mandatory properties or possibly also the recommended ones.
+Additionally, a user can also decide to test their private images, although this isn't a necessity.
+
+### Implementation
+
+The script [`image-md-check.py`](https://github.com/SovereignCloudStack/standards/blob/main/Tests/iaas/image-metadata/image-md-check.py)
```

Comment:
I think we should use a (somewhat) stable permalink here:
```suggestion
The script [`image-md-check.py`](https://github.com/SovereignCloudStack/standards/blob/59683d8fc592d5d27152be7dab32290d4c84316b/Tests/iaas/image-metadata/image-md-check.py)
```

Reply:
I disagree. The links are supposed to be kept alive using our continuous-integration system. I don't want them to point to some super-old version.

Reply:
The CI can't guarantee this as long as the CI is not immutable itself, which it isn't.
Any practice from the last 15 years writing documentation tells me that links can and will rot away.

Regardless, I don't think this is an important enough issues, to warrant a long winded discussion, as everything can be retrieved via some git archeology, if really needed.